homeassistant/components/ukraine_alarm/translations/uk.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "step": {
+            "user": {
+                "data": {
+                    "region": "\u0420\u0435\u0433\u0456\u043e\u043d"
```

Comment:
Just a note that these translations will need to be added to Lokalise, as that’s the source of truth. Next translation update will override these files.

Reply:
Ok, will add them after merge.